code/modules/events/spacevine.dm

```diff
@@ -419,6 +431,7 @@
 	max_integrity = 50
 	var/energy = 0
 	var/can_spread = TRUE //Can this kudzu spread?
+	var/can_tangle = TRUE //Can this kudzu tangle
```

Comment:
```suggestion
	/// Can this kudzu tangle? [Mothblocks note: Please describe what this means, it's not quite obvious to me even though I've seen plenty of kudzu in my time]
	var/can_tangle = TRUE
```

Reply:
The triple-slash comment is for autodocs

Reply:
alright